### ISSUE TITLE
Make scatterers and theory available from holopy.scattering?

### ISSUE THREAD

**Issue by tdimiduk** [manoharan-lab/holopy#21]
Is it worth making theories (Mie, Multisphere, DDA, ...) and scattereres (Sphere, Spheres, ...) available from holopy.scattering instead of just in the theory and scattererer namespaces? 

Example, currently:
``` python
from holopy.scattering.scatterer import Sphere, Spheres
from holopy.scattering.theory import Mie, Multisphere
```

If we pushed these namespaces down, it would then be possible to do
``` python
from holopy.scattering import Sphere, Spheres, Mie, Multisphere
```
though the old code would of course still work.

I don't think we have enough things here fro this namespace to get too crowded, and it would save a line and chuck of typing from some of the most common use cases of holopy.

**Comment by barkls** [comment]
I moved calc_holo() and similar functions into holopy.scattering from holopy.scattering.calculations. I considered moving the scatterers as well but decided not to until chatting about it (I hadn't noticed this issue). If we both independently thought it might be a good idea then I will go ahead and implement it.

**Comment by barkls** [comment]
fixed in most recent commits to pr #79
